src/app/(auth)/login/page.tsx

```diff
@@ -56,20 +42,18 @@ export default function Login() {
             <TextInput
               label="Email"
               placeholder="email@example.com"
-              erroring={false}
-              errorText="Email Error" // {errorMessage}
+              errorText="" // {errorMessage}
               type="email"
-              name="email"
+              id="email"
               value={email}
               setValue={setEmail}
             />
             <TextInput
               label="Password"
               placeholder="Password"
-              erroring={false}
-              errorText="Password Error"
+              errorText=""
               type="password"
-              name="password"
+              id="password"
               value={password}
               setValue={setPassword}
             />
```

Comment:
Github does not let me comment outside of the diff, but:

Can we route the "Forgot your password?" text to the `/forgot-password` route instead? i.e. on line 62

Reply:
yup!